File: src/main/java/com/github/tiniyield/sequences/benchmarks/common/data/providers/rest/countries/Countries.java
package com.github.tiniyield.sequences.benchmarks.common.data.providers.rest.countries;

import com.github.tiniyield.sequences.benchmarks.common.data.loader.FileLoader;
import com.github.tiniyield.sequences.benchmarks.common.model.country.Country;

public class Countries {

    public final Country[] data;

    public Countries() {
        data = new FileLoader().loadCountries().toArray(Country[]::new);
    }

}

File: src/main/kotlin/com/github/tiniyield/sequences/benchmarks/kt/zip/Countries.kt
package com.github.tiniyield.sequences.benchmarks.kt.zip

import com.github.tiniyield.sequences.benchmarks.common.data.providers.rest.countries.Countries
import com.github.tiniyield.sequences.benchmarks.common.model.country.Country

val countriesProvider = Countries()

fun countries(): Sequence<Country> {
    return countriesProvider.data.asSequence()
}
